rest_tools/server/handler.py

```diff
@@ -147,9 +145,11 @@ def get_json_body_argument(
         self,
         name: str,
         default: Any = arghandler.NO_DEFAULT,
+        type_: Optional[type] = None,
```

Comment:
`type_` feels a bit weird to use as part of the API. I know the underscore is to avoid keyword clashes, but it feels like "internal variable brought up to the API level".

Any chance you'd rename it `as_type` or something similar?

Reply:
What do you think about `typ`? I want to keep it a single word

Reply:
or just `type`?

Reply:
If `type` will work, then yeah I think that'd be best. If it doesn't, I think I'd prefer `typee` (think "Yogurt Shopee") to `typ`, but meh, these are all just suggestions. Even `type_` is fine, even if a little weird to me. :smile:

Reply:
`type` it is! It's legal flake8 and argparse uses it, probably because of the same conundrum we had